apps/prairielearn/src/lib/cookie.ts

```diff
@@ -10,3 +10,32 @@ export function shouldSecureCookie(req: Request): boolean {
   // be used when the app is behind a reverse proxy or load balancer.
   return !config.devMode || req.protocol === 'https';
 }
+
+/**
+ * Helper function to clear a cookie regardless of if it was set with an
+ * explicit domain or not.
+ */
+export function clearCookie(res: Response, name: string): void {
```

Comment:
For a consistent-ish interface, we could have this accept an array of old/new names, like `setCookie`?

Reply:
Added in ede3655.

Reply:
Looks good, these helper functions make it very easy to see that we are doing the right thing.